src/js/components/Grommet/index.d.ts

```diff
@@ -8,6 +8,7 @@ export interface GrommetProps {
   cssVars?: boolean;
   dir?: 'rtl';
   full?: boolean | 'min';
+  options?: object;
```

Comment:
I think this should be replaced with the actual expected structure of the object that `options` can accept.